python/ray/tune/execution/tune_controller.py

```diff
@@ -203,8 +203,11 @@ def _cleanup_stopping_actors(self, force_all: bool = False):
         # This is a bit costly, so we want to avoid running it too often
         times = deque(
             sorted(
-                (timestamp, tracked_actor)
-                for tracked_actor, timestamp in self._stopping_actors.items()
+                [
+                    (timestamp, tracked_actor)
+                    for tracked_actor, timestamp in self._stopping_actors.items()
+                ],
+                key=lambda item: item[0],
```

Comment:
This is to never reorder based on "tracked actor"?

Reply:
Yes! This can run into errors when the timestamp is the same